`test_default_filters` integration test fails with `AttributeError: 'NoneType' object has no attribute 'params'` error

Link: https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-10674/1/execution/node/107/log/

Out integration tests CI job fails with the following error:

```
17:20:12  =================================== FAILURES ===================================
17:20:12  _____________________ SctEventsTests.test_default_filters ______________________
17:20:12  
17:20:12  self = <unit_tests.test_events.SctEventsTests testMethod=test_default_filters>
17:20:12  
17:20:12      @pytest.mark.integration
17:20:12      def test_default_filters(self):
17:20:12      
17:20:12          with environment(SCT_CLUSTER_BACKEND='docker'):
17:20:12  >           enable_default_filters(SCTConfiguration())
17:20:12  
17:20:12  unit_tests/test_events.py:413: 
17:20:12  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
17:20:12  sdcm/sct_events/setup.py:157: in enable_default_filters
17:20:12      ignore_topology_change_coordinator_errors().__enter__()
17:20:12  /usr/local/lib/python3.10/contextlib.py:135: in __enter__
17:20:12      return next(self.gen)
17:20:12  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
17:20:12  
17:20:12      @contextmanager
17:20:12      def ignore_topology_change_coordinator_errors():
17:20:12          with ExitStack() as stack:
17:20:12              if SkipPerIssues(
17:20:12                      issues=[
17:20:12                          "https://github.com/scylladb/scylladb/issues/20754",
17:20:12                          "https://github.com/scylladb/scylladb/issues/20950",
17:20:12                      ],
17:20:12  >                   params=TestConfig().tester_obj().params,
17:20:12              ):
17:20:12  E           AttributeError: 'NoneType' object has no attribute 'params'
17:20:12  
17:20:12  sdcm/sct_events/group_common_events.py:82: AttributeError
```

Most probably it is the result of the following PR:
- https://github.com/scylladb/scylla-cluster-tests/pull/10386

--- a/sdcm/sct_events/group_common_events.py
+++ b/sdcm/sct_events/group_common_events.py
@@ -100,12 +100,6 @@ def ignore_topology_change_coordinator_errors():
             ))
             stack.enter_context(DbEventsFilter(
                 db_event=DatabaseLogEvent.RUNTIME_ERROR,
-                line=r".*raft_topology - topology change coordinator fiber got error std::runtime_error"
-                     r" \(raft topology: exec_global_command\(barrier\) failed with seastar::rpc::closed_erro"
-                     r"r \(connection is closed\)\)"
-            ))
-            stack.enter_context(DbEventsFilter(
-                db_event=DatabaseLogEvent.RUNTIME_ERROR,
                 line=r".*raft_topology - drain rpc failed, proceed to fence old writes:.*connection is closed",
             ))
         yield

--- a/sdcm/sct_events/setup.py
+++ b/sdcm/sct_events/setup.py
@@ -33,7 +33,6 @@ from sdcm.sct_events.events_processes import \
     EVENTS_GRAFANA_ANNOTATOR_ID, EVENTS_GRAFANA_AGGREGATOR_ID, EVENTS_GRAFANA_POSTMAN_ID, \
     EventsProcessesRegistry, create_default_events_process_registry, get_events_process, EVENTS_HANDLER_ID, EVENTS_COUNTER_ID
 from sdcm.utils.issues import SkipPerIssues
-from sdcm.sct_events.group_common_events import ignore_topology_change_coordinator_errors
 
 
 EVENTS_DEVICE_START_DELAY = 1  # seconds
@@ -154,7 +153,10 @@ def enable_default_filters(sct_config: SCTConfiguration):  # pylint: disable=unu
     # upgrades and any place where the race between raft global barrier and gossipier could
     # take place. So ignore such messages globally for any sct test.
     # TODO: this should be removed after gossiper will be removed.
-    ignore_topology_change_coordinator_errors().__enter__()
+    DbEventsFilter(db_event=DatabaseLogEvent.RUNTIME_ERROR,
+                   line=r".*raft_topology - topology change coordinator fiber got error std::runtime_error"
+                        r" \(raft topology: exec_global_command\(barrier\) failed with seastar::rpc::closed_error"
+                        r" \(connection is closed\)\)").publish()
 
 
 __all__ = ("start_events_device", "stop_events_device", "enable_default_filters")

--- a/unit_tests/test_events.py
+++ b/unit_tests/test_events.py
@@ -412,7 +412,7 @@ class SctEventsTests(BaseEventsTest):  # pylint: disable=too-many-public-methods
         with environment(SCT_CLUSTER_BACKEND='docker'):
             enable_default_filters(SCTConfiguration())
 
-        with self.wait_for_n_events(self.get_events_logger(), count=4):
+        with self.wait_for_n_events(self.get_events_logger(), count=5):
             DatabaseLogEvent.BACKTRACE() \
                 .add_info(node="A",
                           line_number=22,
@@ -439,6 +439,13 @@ class SctEventsTests(BaseEventsTest):  # pylint: disable=too-many-public-methods
                      ".c.sct-project-1.internal/10.142.1.155:9042: Cannot achieve consistency level for cl ONE. Requires 1, alive 0",
             ).publish()
 
+            DatabaseLogEvent.RUNTIME_ERROR().add_info(
+                node="A",
+                line_number=22,
+                line="ERROR 2023-12-18 12:45:25,673 [shard 0: gms] raft_topology - topology change coordinator fiber got error std::runtime_error "
+                     "(raft topology: exec_global_command(barrier) failed with seastar::rpc::closed_error (connection is closed))"
+            ).publish()
+
         log_content = self.get_event_log_file("events.log")
 
         self.assertIn("other back trace", log_content)
@@ -451,6 +458,7 @@ class SctEventsTests(BaseEventsTest):  # pylint: disable=too-many-public-methods
         error_log_content = self.get_event_log_file("error.log")
         assert 'data_dictionary::no_such_column_family' not in error_log_content
         assert 'Authentication error' in error_log_content
+        assert 'topology change coordinator fiber got error' not in error_log_content
 
     def test_failed_stall_during_filter(self):
         with self.wait_for_n_events(self.get_events_logger(), count=5, timeout=3):

